purchase_request/views/purchase_request_view.xml

```diff
@@ -0,0 +1,406 @@
+                            <field
+                                name="description"
+                                attrs="{'readonly': [('is_editable','=', False)]}"
+                            />
+                            <field
+                                name="group_id"
+                                attrs="{'readonly': [('is_editable','=', False)]}"
+                            />
+                        </group>
+                        <group>
+                            <field
+                                name="date_start"
+                                attrs="{'readonly': [('is_editable','=', False)]}"
+                            />
+                            <field
+                                name="picking_type_id"
+                                attrs="{'readonly': [('is_editable','=', False)]}"
+                            />
+                            <field
+                                name="company_id"
+                                groups="base.group_multi_company"
+                                widget="selection"
+                                attrs="{'readonly': [('is_editable','=', False)]}"
+                            />
+                        </group>
+                    </group>
+                    <notebook>
+                        <page string="Products">
+                            <field name="line_ids">
+                                <tree
+                                    decoration-muted="cancelled == True"
+                                    editable="bottom"
+                                >
+                                    <field name="product_id" />
+                                    <field name="name" />
+                                    <field name="product_qty" />
+                                    <field
+                                        name="product_uom_id"
+                                        groups="uom.group_uom"
+                                    />
+                                    <field
+                                        name="analytic_account_id"
+                                        groups="analytic.group_analytic_accounting"
+                                    />
+                                    <field
+                                        name="analytic_tag_ids"
+                                        groups="analytic.group_analytic_accounting"
+                                    />
+                                    <field name="date_required" />
+                                    <field name="estimated_cost" widget="monetary" />
+                                    <field name="currency_id" invisible="1" />
+                                    <field
+                                        name="company_id"
+                                        groups="base.group_multi_company"
+                                        widget="selection"
+                                    />
```

Comment:
Replace for preferred_supplier
company_id field is a related readonly from the order, it does not give additional information.

Reply:
This fields doesn't exist in the code, you mean create it?

Reply:
supplier_id technical name Preferred Supplier as String name

https://github.com/OCA/purchase-workflow/pull/1342/files#diff-3ce1fbd00b838bb7ff1423c460773e0a8c5bb02b2e31a945fdbd132cb8282f11R89-R95